enterprise/plugins/com.liferay.ide.studio.ui/plugin.xml

```diff
@@ -69,11 +69,11 @@
         point="org.eclipse.ui.splashHandlers">
         <splashHandler
             class="com.liferay.ide.studio.ui.StudioSplashHandler"
-            id="com.liferay.ide.studio.ui.splashHandler">
+            id="com.liferay.ide.porodcut.splashHandler">
```

Comment:
This shoudl be `product` right?  can you change and retest and if it is ok, resend?  thanks